.eslintrc

```diff
@@ -260,6 +260,6 @@
     "import/no-duplicates": 2,
     "import/newline-after-import": 2,
     "import/unambiguous": 2,
-    "import/extensions": [ 1, "always" ]
+    "import/extensions": [ 0, "always" ]
```

Comment:
Tried that, still the same. I even cleared `node_modules` and did a fresh `npm install`